src/features/coursesFeatures/Courses/Courses.tsx

```diff
@@ -0,0 +1,36 @@
+import { CourseCard } from '@/widgets';
+import './Courses.scss';
```

Comment:
There is no need in putting "*Features" as a name for each feature - since it is in the "features" folder, it should be obvious :)